SuperblocksHQ/ethereum-studio
# ES-64 - Display preview of md files

### Description of the Change
Adds new functionality so users can see markdown files in preview mode. By default, when markdown file is opened it shows preview, to toggle source code, there is a new button in toolbar (right under the file tabs).

![Screenshot from 2019-10-28 18-19-02](https://user-images.githubusercontent.com/17637244/67674573-717b2680-f9af-11e9-8c5e-301748da83bf.png)



### Benefits
Better UX



### Verification Process
0. Run `npx lerna bootstrap --hoist` to install new added package `showdown`
1. Open `README.md` file or any other markdown file
2. Markdown should render correctly
3. Toggle source code and edit markdown
4. Toggle back to preview and check updated preview


### Github Issues
Resolves #34
